pdo-connection.php

```diff
@@ -1,20 +1,27 @@
 <?php
 
-$host = 'db';
-$user = 'root';
-$password = 'password';
-$charset = 'utf8mb4';
+const HOST = 'db';
+const USER = 'root';
+const password = 'password';
```

Comment:
password should be written in caps - we always make consts caps to make them really obvious

Reply:
Changed